client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1672,12 +1672,16 @@
                 function runSendAndContinue() {
```

Comment:
Maybe consider adding a docstring, though I don't know whether we write them for helper functions?
But it's probably a good idea nevertheless, since the function is quite long...